src/test/java/duke/parser/ParserTest.java

```diff
@@ -0,0 +1,26 @@
+package duke.parser;
+
+import duke.data.exception.DukeException;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+public class ParserTest {
+    @Test
+    public void parser_getInstruction_success() {
+        assertEquals("todo", new Parser().getInstruction("todo this"));
+    }
+
+    @Test
+    public void parser_getDeadlineDetail_success() throws DukeException {
+        assertEquals("this", new Parser().getDeadlineDescription("deadline this /by now"));
```

Comment:
The code mainly has mentions of "description", but in the test method name, it is referred to as "detail". Should the test method name be updated?